packages/node-opentelemetry/src/otel.ts

```diff
@@ -134,17 +140,40 @@ export const initSDK = (config: SDKConfig) => {
     );
   }
 
+  hdx(
+    stopOnTerminationSignals
```

Comment:
can we also add this to the init log ? [line 103](https://github.com/hyperdxio/hyperdx-js/blob/58ab2dd39f6e1a2b91d417367cc8a9973f9b1de9/packages/node-opentelemetry/src/otel.ts#L103))